cluster/gce/gci/README.md

```diff
@@ -21,12 +21,36 @@ Policy](https://cloud.google.com/container-optimized-os/docs/resources/support-p
 Container-Optimized OS images are used by kubernetes [End-to-End tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) and
 [Node End-to-End tests](https://github.com/kubernetes/community/tree/master/contributors/devel/sig-node). To see current
 active releases, please refer to COS's [Release
-Notes](https://cloud.google.com/container-optimized-os/docs/release-notes). When
-choose images for testing, the latest LTS images is suggested for running
-presubmit, postsubmit or periodics tests since the image is stable and has
-latest bug and security fixes. For testing requiring new features or continuous integration,
-the latest dev or beta or stable channel can be used.
-are needed, the latest LTS images or stables are preferred.
+Notes](https://cloud.google.com/container-optimized-os/docs/release-notes).
+
+### How to choose image for your test
+
+  * To run presubmit, postsubmit or periodic tests, the latest LTS images are
+    preferred. If tests need two images, you can use the latest two LTS images.
+    For example, 81 and 77. LTS images are stable and usually include latest bug and security fix.
+
+  * To try out new features or integrate continuously with other container
+    related technologies like runc, containerd, docker and kubernertes, the
+    latest dev or beta or stable images are preferred.
+
+  * To update the images, using image in the same channel is preferred. However,
+    69 is the first LTS image. Before that, COS only has dev, beta and stable
+    images. That is why stable images are used quite frequently in current testing.
+    For now, images should slowly migrate from stable to LTS if possible. For
+    testing using dev ore beta, we need to consider the original intention and
+    keep using image in existing channel unless we understand the beneath reason.
+
+  * There are three ways to update image used by each testing suite: image,
+    image_regex or image_family. image is preferred, but it needs frequent
+    change and may use deprecated image when latest COS image is released. image_regex can
+    be used if you want to image with the same naming pattern, while
+    image_family can also be used if you always want to use latest image in the
+    same family. Howevere, if something went wrong, it will hard to rollback
+    images using image_regex and image_family. One option is to
+    update yaml files automatically. When things do not come
+    as expected, revert should avoid the surprise.
```

Comment:
This paragraph might be better in some overall e2e_node testing document.

I think we need to sort of discuss that images might have multiple names.

Going from memory:
sometimes we had cos-77-stable-#####-##
but really that would have been the same as cos-77-#####-##
And we should prefer to use the one WITHOUT stable, as it won't always be marked as 'stable', even if it or a later version goes to lts.